### PR TITLE
update README

### PR DESCRIPTION
**Pull request by srinivasupadhya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,67 +1,41 @@
...
 
 ## To Dos
 - Clean up the code esp. the JSON SerDe part
 - Add proper tests around the plugin
-- Add support for [Github's commit status](https://developer.github.com/v3/repos/statuses/) API to push build status to Github. May be a separate task plugin?
+- Add support for [Github's commit status](https://developer.github.com/v3/repos/statuses/) API to push build status to Github. May be a separate task/notification plugin?
 
 ## FAQs
 
-### How Pull Requests are Built?
```
Comment (ashwanthkumar) [inline README.md]: This doesn't help is it?

Comment (ashwanthkumar) [inline README.md]: May be not.

File: README.md [hunk 2]
```diff
@@ -1,67 +1,41 @@
...
-We periodically poll for new PRs on the given repository. We build a pull request when it's first opened and when commits are added to the pull request throughout its lifetime.
-Rather than test the commits from the branches the pull request is sent from, we test the merge between the origin and the upstream branch.
-
-### Pull Request isn't being built
```
Comment (ashwanthkumar) [inline README.md]: But but this one?

Comment (srinivasupadhya) [inline README.md]: i added behavior section explaining it. you can keep the FAQ to add any corner cases that might arise.

Comment (srinivasupadhya) [inline README.md]: Also, now that we dont merge before build this case might not arise?

Comment (ashwanthkumar) [inline README.md]: We never used to merge on our side, we take the PR/merge/ commit that Github creates. We still do only that, isn't? The `PR_MERGE_PREFIX` actually points to the merge commits on the PRs. Please help me understand.

**Comment by srinivasupadhya** [comment]
I reverted FAQ changes. I have put back yours (the 2nd question since the 1st is explained in behavior).

**Comment by srinivasupadhya** [comment]
reference - https://github.com/srinivasupadhya/gocd-build-github-pull-requests/tree/readme

**Pull request merged**